Docs/reference/content/what_is_new.md

```diff
@@ -8,6 +8,12 @@ title = "What's New"
   pre = "<i class='fa fa-star'></i>"
 +++
 
+## What's New in 2.10.0-beta1
+Some of the changes in 2.10.0-beta1 include:
+
+* Client-side field level encryption support for Windows
+
+
```

Comment:
Does the extra blank line serve a purpose?

Reply:
Removed!